pkg/cableengine/ipsec/strongswan_test.go

```diff
@@ -0,0 +1,66 @@
+package ipsec
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/kelseyhightower/envconfig"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Util", func() {
+	Describe("Charon port configuration", testCharonConfigPortsGen)
+})
+
+func testCharonConfigPortsGen() {
+	Context("config rendering", func() {
+		It("should render config correctly out of strongSwan parameters", func() {
```

Comment:
add the check that the  FD is closed after

Reply:
Yup, I'll check how to do that...

Reply:
I see no decent way of checking that properly, I accept ideas around how to test that on writeCharonConfig, although I believe that belongs more to funcitonal testing, that we don't have currently, and that's why I broke the render part, to be able to test it.